### ISSUE TITLE
Fix release workflow

### ISSUE THREAD

**Issue by nrempel** [nrempel/sleek#2]
https://github.com/nrempel/sleek/blob/main/.github/workflows/release.yml

This workflow should build a release binary for every platform and make it available as a Github release when a version tag is pushed. It's not working right now.

**Comment by balling-dev** [comment]
Thanks for `sleek`.

In my [fork](https://github.com/balling-dev/sleek), I added a working release workflow using `cargo dist` (see [docs](https://opensource.axo.dev/cargo-dist/book/way-too-quickstart.html)).

It creates and uploads binaries for Linux, Mac and Windows to a github release when a commit is tagged with `vX.Y.X` (note that it does *not* publish to crates.io). 

I will be happy to create a pull request if that is helpful :smiley: 

Br,

./ Kristoffer

**Comment by nrempel** [comment]
Yes, thank you! Please feel free to up a pull request. That will be very helpful. :)

**Comment by nrempel** [comment]
Thanks for this—I've done some work to fix the workflow. A release with binaries is now produced for tags.

https://github.com/nrempel/sleek/releases/tag/v0.4.0
